checkpoint3/utils.cs

```diff
@@ -2,38 +2,54 @@
 using System.Linq;
 using System.Collections;
 using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore;
+
 
 namespace checkpoint3
 {
-    class Utils
+    class Utils : DbContext
     {
+        public Utils(DbContextOptions<Utils> options) : base(options)
+        {
+
+        }
         public List<ToDo> ToDos = new List<ToDo>();
```

Comment:
This is almost right. You have to declare your ToDos property as a type of DbSet<ToDo> and it needs to be a property not a field.